src/Compilers/CSharp/Portable/Lowering/SyntheticBoundNodeFactory.cs

```diff
@@ -612,6 +612,16 @@ public BoundCall Call(BoundExpression receiver, MethodSymbol method, ImmutableAr
             { WasCompilerGenerated = true };
         }
 
+        public BoundCall Call(BoundExpression receiver, MethodSymbol method, ImmutableArray<RefKind> argumentRefKinds, ImmutableArray<BoundExpression> args)
+        {
+            Debug.Assert(method.ParameterCount == args.Length);
+            return new BoundCall(
+                Syntax, receiver, method, args,
+                ImmutableArray<String>.Empty, argumentRefKinds, false, false, false,
+                ImmutableArray<int>.Empty, LookupResultKind.Viable, method.ReturnType)
```

Comment:
Thanks!

In copying the form of the method above, I copied the (apparently incorrect) value of the argsToParams mapping. This lead me to change the local rewriter to treat empty arrays as equivalent to arrays with default values, figuring that I was fixing a bug.